pkg/endpoint/local_ip.go

```diff
@@ -32,6 +32,14 @@ var (
 		k8snet.IPv4: "8.8.8.8",
 		k8snet.IPv6: "2001:4860:4860::8888",
 	}
+
+	// These addresses are used internally by OVN-Kubernetes CNI
+	// and should not be used as local IPs.
+	infraInternalIPs = map[string]struct{}{
+		"fd69::2":       {},
+		"169.254.169.2": {},
+	}
```

Comment:
Could use a `Set` here.

```suggestion
        import "k8s.io/apimachinery/pkg/util/sets"
        ...
        
	infraInternalIPs  := sets.New("fd69::2", "169.254.169.2")
```

Reply:
+1 but please use `k8s.io/utils` sets rather than `apimachinery`.